pkg/operator/operatorclient/interfaces.go

```diff
@@ -0,0 +1,10 @@
+package operatorclient
+
+const (
+	EtcdNamespaceName                     = "kube-system"
+	KubeAPIServerNamespaceName            = "openshift-kube-apiserver"
+	UserSpecifiedGlobalConfigNamespace    = "openshift-config"
```

Comment:
GlobalUserSpecifiedConfigNamespace. The config is user specified, not the namespace.

Reply:
> GlobalUserSpecifiedConfigNamespace. The config is user specified, not the namespace.

will do either later or if I don't go green.